networks/tiny_yolo_v1/Makefile

```diff
@@ -9,10 +9,10 @@ NOCOLOR = '\033[0m'
 MODEL_BASE_NAME = tiny-yolo-v1_53000
 
 PROTOTXT_FILENAME= tiny-yolo-v1.prototxt
-GET_PROTOTXT = wget --no-cache -P . http://ncs-forum-uploads.s3.amazonaws.com/ncappzoo/tiny_yolo/${PROTOTXT_FILENAME}
+GET_PROTOTXT = wget --no-cache -P . https://downloadmirror.intel.com/29611/eng/tiny_yolo_v1_caffe.zip${PROTOTXT_FILENAME}
```

Comment:
Since the prototxt and caffemodel files are being downloaded in a single zip file, You can consolidate both the GET_PROTOTXT  (line 12) and GET_CAFFEMODEL (line 15) into a single variable. Maybe something like GET_MODEL_ZIP. Also you don't need to include the $CAFFEMODEL_FILENAME and $PROTOTXT_FILENAME at the end of the new wget command because we have already specified the zip file name.

Reply:
Also I think maybe we could remove the [prototxt](https://github.com/movidius/ncappzoo/blob/master/networks/tiny_yolo_v1/Makefile#L35) and [caffemodel](https://github.com/movidius/ncappzoo/blob/master/networks/tiny_yolo_v1/Makefile#L53) make targets since we are not downloading them individually. Otherwise as it currently is, it will download the zip file twice. 

In their place, we can probably add a "get_model" make target which will check if the model files are present and run  GET_MODEL_ZIP (from above) and unzips the zip file if the model files do not exist.